ophidian/src/ophidian/__main__.py

```diff
@@ -0,0 +1,91 @@
+# coding=utf-8
+#
+# This file is part of Hypothesis, which may be found at
+# https://github.com/HypothesisWorks/hypothesis-python
+#
+# Most of this work is copyright (C) 2013-2017 David R. MacIver
+# (david@drmaciver.com), but it contains contributions by others. See
+# CONTRIBUTING.rst for a full list of people who may hold copyright, and
+# consult the git log if you need to determine who owns an individual
+# contribution.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public License,
+# v. 2.0. If a copy of the MPL was not distributed with this file, You can
+# obtain one at http://mozilla.org/MPL/2.0/.
+#
+# END HEADER
+
+from __future__ import division, print_function, absolute_import
+
+import os
+import sys
+
+import click
+from ophidian.core import Ophidian, NoSuchPython
+from ophidian.finder import default_paths
+from ophidian.storage import DirStorage
+from ophidian.installer import PyenvInstaller
+
+
+@click.command()
+@click.option('--major', type=int, default=None)
+@click.option('--minor', type=int, default=None)
+@click.option('--micro', type=int, default=None)
+@click.option('--implementation', type=str, default=None)
+@click.option('--identifier', type=str, default=None)
+@click.option('--install/--no-install', default=True)
+def main(major, minor, micro, install, implementation, identifier):
+    homedir = os.environ['OPHIDIAN_HOME']
+
+    builds = os.path.join(homedir, 'builds')
+
+    try:
+        os.makedirs(builds)
+    except FileExistsError:
```

Comment:
This exception doesn’t exist on Python 2. And if you're choosing to go Python 3 only, then you might prefer

```python
os.makedirs(builds, exist_ok=True)
```